sessionctx/variable/sysvar.go

```diff
@@ -351,15 +351,6 @@ var defaultSysVars = []*SysVar{
 		s.AllowAggPushDown = TiDBOptOn(val)
 		return nil
 	}},
-	{Scope: ScopeGlobal | ScopeSession, Name: TiDBOptBCJ, Value: BoolToOnOff(DefOptBCJ), Type: TypeBool, Validation: func(vars *SessionVars, normalizedValue string, originalValue string, scope ScopeFlag) (string, error) {
-		if TiDBOptOn(normalizedValue) && vars.AllowBatchCop == 0 {
-			return normalizedValue, ErrWrongValueForVar.GenWithStackByArgs(TiDBOptBCJ, "'true' while tidb_allow_batch_cop is 0, please active batch cop at first.")
-		}
-		return normalizedValue, nil
```

Comment:
Can you add an entry into here: https://github.com/pingcap/tidb/blob/master/sessionctx/variable/removed.go

This helps provide some backward compatibility in case applications required this setting.